docs/network protocol.md

```diff
@@ -147,13 +148,12 @@ Sent to clients to acknowledge a received [LocationScouts](#LocationScouts) pack
 
 ### RoomUpdate
 Sent when there is a need to update information about the present game session. Generally useful for async games.
-Once authenticated (received Connected), this may also contain data from Connected.
 #### Arguments
-The arguments for RoomUpdate are identical to [RoomInfo](#RoomInfo) barring:
+RoomUpdate can contain arguments from [RoomInfo](#RoomInfo) and, once authenticated, arguments from
+[Connected](#Connected) barring:
```

Comment:
Wonder if this whole section should just be rewritten to be more clear. Interesting that we include the whole table with descriptions below when it's basically saying, "Yeah, these aren't in this packet."